validator/include/sim_validator.h

```diff
@@ -24,30 +24,28 @@
  * WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
  */
 
-#ifndef RENODE_VALIDATOR_H
-#define RENODE_VALIDATOR_H
+#ifndef SIM_VALIDATOR_H
+#define SIM_VALIDATOR_H
 
 #include "validator.h"
-#include "renode_interface.h"
```

Comment:
Can `policy-engine/validator/include/renode_interface.h` be deleted as a result of this change?

Reply:
done